app/queries/schools/with_ects_with_no_mentor_query.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Schools
+  class WithEctsWithNoMentorQuery
```

Comment:
Can we make this inherit from `::BaseService` then we get the ability to `.call` instead of `.new.call`?